src/java/com/palantir/cassandra/utils/SerializablePair.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.cassandra.utils;
+
+import java.io.Serializable;
+
+public final class SerializablePair<T1, T2> implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    private final T1 key;
+    private final T2 value;
+
+    public SerializablePair(T1 key, T2 value) {
+        this.key = key;
+        this.value = value;
+    }
+
+    public T1 getKey() {
+        return key;
+    }
+
+    public T2 getValue() {
+        return value;
+    }
+
+    public static <X, Y> SerializablePair<X, Y> create(X x, Y y) {
+        return new SerializablePair<>(x, y);
+    }
+}
```

Comment:
Feels overkill, if ColumnFamilyStore is unfortunately not serializable, we can probably have two functions:
```
public Map<String, Integer> getCompactionsPendingTasksByCf()

public Map<ColumnFamilyStore, Integer> getCompactionsPendingTasks()
```
with the first one calling the second one and using the format you initially defined